packages/react-core/src/components/MultipleFileUpload/MultipleFileUploadStatusItem.tsx

```diff
@@ -45,6 +45,8 @@ export interface MultipleFileUploadStatusItemProps extends React.HTMLProps<HTMLL
   progressAriaLabel?: string;
   /** Associates the ProgressBar with it's label for accessibility purposes. Required when title not used */
   progressAriaLabelledBy?: string;
+  /** Unique identifier for Progress. Generated if not specified. */
```

Comment:
nit.  lowercase for progress here.  
It also looks like the id is not generated.

Reply:
It doesn't get generated in this file, but it does inside of `Progress`, where we pass `progressId` to the `id` prop and this the logic in that component: https://github.com/patternfly/patternfly-react/blob/ea14d7d9c04f63e62c8811090c5107edc18aa4d8/packages/react-core/src/components/Progress/Progress.tsx#L67.

I could rephrase and say something like `Generated inside of Progress if not specified.`?

In terms of the case of `progress`, I assumed it to be capitalized as it's referring to the component, just as the above comment on line 46 refers to `ProgressBar` in similar TitleCase.

Reply:
The above should probably be in sentence case too, "progress bar".
I think the comment is fine as is for the progressId.